stmt.go

```diff
@@ -150,18 +160,24 @@ func FetchRowCount(rowCount int) Option { return FetchArraySize(rowCount) }
 
 // FetchArraySize returns an option to set the rows to be fetched, overriding DefaultFetchRowCount.
 func FetchArraySize(rowCount int) Option {
```

Comment:
For node-oracledb, we throw a node-oracledb error when the arraysize is 0.  This is a little nicer than a DPI error because it can give the specific parameter name.  Is that something to think about for godror?

Reply:
Zero array size is uninterpretable, so leave 0 as default, negative is an error (or default).